clean_code_main/clean_code_checks/y_check_aunit_coverage_check.clas.abap

```diff
@@ -0,0 +1,375 @@
+class Y_CHECK_AUNIT_COVERAGE_CHECK definition
+  public
+  inheriting from Y_CHECK_BASE
+  create public .
+
+public section.
+
+  methods CONSTRUCTOR .
+
+  methods GET_ATTRIBUTES
+    redefinition .
+  methods IF_CI_TEST~QUERY_ATTRIBUTES
+    redefinition .
+  methods PUT_ATTRIBUTES
+    redefinition .
+  methods RUN
+    redefinition .
```

Comment:
why do you redefine the 3 methods of the base class?

i would understand, execute_check( ) & inspect_tokens( ).

Reply:
It appears, that the existing Code Pal checks are all inheriting from the main class Y_CHECK_BASE and by that using the same parameter screen with ("Consider Objects created after", "Message severity" etc...). When creating my class, I did not had this main class and created a custom parameter screen. If the usage of the parameters is mandatory, I can include them to my check. 
However, I would still need own parameters for the required coverage and therefore redefine GET/PUT/QUERY. I would rather not have those parameters as fixed values inside of the code, since the minium code coverage might be defined differently from project to project: In my current project I defined 60% as minimum, others software projects are expecting higher/lower coverages.